src/app/api/mock/dataset.mock.ts

```diff
@@ -231,6 +231,7 @@ export const MOCK_NODES = [
             isRoot: true,
             isCurrent: true,
             color: "#7aa3e5",
+            account: "kamu",
```

Comment:
Let's call it `accountName` for consistency with the API.

Reply:
Fixed